homeassistant/components/minecraft_server/quality_scale.yaml

```diff
@@ -0,0 +1,113 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Integration doesn't provide any service actions.
+  appropriate-polling:
+    status: done
+    comment: Default polling interval is set to 60s.
+  brands: done
+  common-modules: done
+  config-flow: done
+  config-flow-test-coverage: done
```

Comment:
`test_java_connection`, let's not directly add the `data` to the user step and first receive the first step and then configure the second using that data